client/src/bundles/ConfirmFolder.js

```diff
@@ -138,13 +138,11 @@ jQuery.entwine('ss', ($) => {
       const handleSubmit = (...args) => this._handleSubmitModal(...args);
       const title = i18n._t('UserForms.FILE_CONFIRMATION_TITLE', 'Select file upload folder');
       const editableFileFieldID = $(this).data('id');
-      const userFormID = $('#Form_EditForm_ID').attr('value');
 
       // Build schema url
-      const parsedURL = url.parse('UserDefinedFormController/confirmfolderform');
+      const parsedURL = url.parse('UserDefinedFormController/confirmfolderformschema');
```

Comment:
I would like to get this link pass via configuration like other form schema URL ideally.